x-pack/elastic-agent/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -200,6 +200,15 @@ func (c *enrollCmd) Execute(ctx context.Context) error {
 	// will communicate to the internal fleet server on localhost only.
 	// Connection setup should disable proxies in that case.
 	localFleetServer := c.options.FleetServer.ConnStr != ""
+	if localFleetServer {
+		token, err := c.fleetServerBootstrap(ctx)
+		if err != nil {
+			return err
```

Comment:
nit: wrap an error with some context?

Reply:
errors are already wrapped with information inside of the `fleetServerBootstrap`.